### PR TITLE
Updated release process to include checking on recent results of Jenkins tests prior to starting the release [BA-6551]

### PR DESCRIPTION
**Pull request by grsterin**

### CODE REVIEW

File: processes/release_processes/README.MD [hunk 1]
```diff
@@ -7,11 +7,16 @@ See our "updating the process" [process](../README.MD)!
...
-If you're the lucky "Release Rotation Engineer" this time, you should do the following three things:
-
-First: Run the publish script to create a new version of Cromwell.
-Second: Run through the "How to Release Cromwell into Firecloud" process.
-Third: Run through the "How to Deploy Cromwell in CAAS prod" process.
+If you're the lucky "Release Rotation Engineer" this time, you should do the following four things:
+
+1. Check that all most recent executions of Cromwell's Jenkins tests have succeeded, and that all those executions were 
+against the most recent commit in "develop" branch. There are 2 groups of Jenkins tests which have to be checked (you 
+should be Under VPN to open the following links):
```
Comment (mcovarr) [inline processes/release_processes/README.MD:14]:
```suggestion
will need to be on the VPN to open the following links):
```

File: processes/release_processes/README.MD [hunk 2]
```diff
@@ -7,11 +7,16 @@ See our "updating the process" [process](../README.MD)!
...
+    1. [Tests for various backends supported by Cromwell](https://gotc-jenkins.dsp-techops.broadinstitute.org/view/cromwell-cron/job/cromwell-cron-parent/)
+    1. [Tests for Cromwell in Terra environment](https://fc-jenkins.dsp-techops.broadinstitute.org/view/Batch/)  
+1. [Run the publish script to create a new version of Cromwell](#how-to-publish-a-new-cromwell-version)  
+1. [Run through the "How to Release Cromwell into Firecloud" process](#how-to-release-cromwell-into-firecloud)  
+1. [Run through the "How to Deploy Cromwell in CAAS prod" process](#how-to-deploy-cromwell-in-caas-prod)
```
Comment (mcovarr) [inline processes/release_processes/README.MD:19]: TOL all the numbers being "1." looks really weird in the source but it does render correctly... 🙂

**Review by mcovarr — APPROVED** [review]

File: processes/release_processes/README.MD [hunk 3]
```diff
@@ -7,11 +7,16 @@ See our "updating the process" [process](../README.MD)!
...
 
-If you're the lucky "Release Rotation Engineer" this time, you should do the following three things:
-
-First: Run the publish script to create a new version of Cromwell.
-Second: Run through the "How to Release Cromwell into Firecloud" process.
-Third: Run through the "How to Deploy Cromwell in CAAS prod" process.
+If you're the lucky "Release Rotation Engineer" this time, you should do the following four things:
+
+1. Check that all most recent executions of Cromwell's Jenkins tests have succeeded, and that all those executions were 
+against the most recent commit in "develop" branch. There are 2 groups of Jenkins tests which have to be checked (you
```
Comment (cjllanwarne) [inline processes/release_processes/README.MD:13]: >against the most recent commit in "develop" branch

This implies an overnight freeze of `develop` prior to running a release. Are we cool with that?

File: processes/release_processes/README.MD [hunk 4]
```diff
@@ -7,11 +7,16 @@ See our "updating the process" [process](../README.MD)!
...
-
-First: Run the publish script to create a new version of Cromwell.
-Second: Run through the "How to Release Cromwell into Firecloud" process.
-Third: Run through the "How to Deploy Cromwell in CAAS prod" process.
+If you're the lucky "Release Rotation Engineer" this time, you should do the following four things:
+
+1. Check that all most recent executions of Cromwell's Jenkins tests have succeeded, and that all those executions were 
+against the most recent commit in "develop" branch. There are 2 groups of Jenkins tests which have to be checked (you 
+will need to be on the VPN to open the following links):
+    1. [Tests for various backends supported by Cromwell](https://gotc-jenkins.dsp-techops.broadinstitute.org/view/cromwell-cron/job/cromwell-cron-parent/)
```
Comment (cjllanwarne) [inline processes/release_processes/README.MD:15]: This link gives me a 404. Does it need to be shared?

Comment (grsterin) [reply processes/release_processes/README.MD:15]: Are you using VPN?

File: processes/release_processes/README.MD (hunk 3 above)
Comment (grsterin) [reply processes/release_processes/README.MD:13]: I would say yes. @mcovarr, what do you think ?

Comment (mcovarr) [reply processes/release_processes/README.MD:13]: 👍

File: processes/release_processes/README.MD (hunk 4 above)
Comment (cjllanwarne) [reply processes/release_processes/README.MD:15]: Yes

**Review by cjllanwarne — APPROVED** [review]

**Pull request merged**